frontend/src/pages/MapView.tsx

```diff
@@ -2,11 +2,7 @@ import React, { useEffect } from 'react';
 import Map, { MapViewportChangeEvent } from '../components/maps/leaflet/Map';
 import './MapView.scss';
 import { getFetchLookupCodeAction } from 'actionCreators/lookupCodeActionCreator';
-import {
-  fetchParcels,
-  fetchPropertyDetail,
-  fetchParcelDetail,
-} from 'actionCreators/parcelsActionCreator';
+import { fetchParcels, fetchPropertyDetail } from 'actionCreators/parcelsActionCreator';
```

Comment:
Should this be also renamed to `fetchProperties, fetchPropertyDetail` and `actionCreators/propertyActionCreator`?

Reply:
See mu previous comment